Validation.dll does not have a NeutralResourcesLanguageAttribute on it

Am getting this now having updated UWP project to 2.4.10:
...\.nuget\packages\Validation\2.4.10\lib\netstandard1.3\Validation.dll : warning MSB3817: The assembly "...\.nuget\packages\Validation\2.4.10\lib\netstandard1.3\Validation.dll" does not have a NeutralResourcesLanguageAttribute on it. To be used in an app package, portable libraries must define a NeutralResourcesLanguageAttribute on their main assembly (ie, the one containing code, not a satellite assembly).

When running unit tests that assert exceptions that Validation throws I now get these failures:
Result Message:	
Assert.Throws() Failure
Expected: typeof(System.ArgumentException)
Actual:   typeof(System.Resources.MissingManifestResourceException): Unable to load resources for resource file "Validation.Strings" in package "8c4135be-e11e-423c-829b-76933c7da27f".

--- a/src/Validation/AssemblyInfo.cs
+++ b/src/Validation/AssemblyInfo.cs
@@ -1,0 +1,6 @@
+﻿// Copyright (c) Andrew Arnott. All rights reserved.
+// Licensed under the Ms-PL license. See LICENSE.txt file in the project root for full license information.
+
+using System.Resources;
+
+[assembly: NeutralResourcesLanguage("en-US")]

